app/contexts/blocks/list_collector.py

```diff
@@ -0,0 +1,87 @@
+from flask import url_for
+from flask_babel import lazy_gettext as _
```

Comment:
Does importing as `_` not trigger any lint errors?? This is a terrible choice of import name for a package

Reply:
The was copied from the original code in `view_context.py`, and is done in more places. It seems reasonable to change it in this pull request. I've now updated all references to gettext in the code with the full method names.

Reply:
`gettext` is commonly aliased as `_()` according to docs, so I think it was fine as is. But can see why some won't like it.

Reply:
@MebinAbraham - this could cause issues in a number of cases, for instance if you wanted to ignore a return value, which is the most common pattern for using `_`